### ISSUE TITLE
Get Rust Nightly running

### ISSUE THREAD

**Issue by darleybarreto** [paugier/nbabel#9]
Hi @paugier, I think it's better discuss about this here than [Julia's discourse](https://discourse.julialang.org/t/nbabel-nbody-integrator-speed-up/51712). You said
> Oh I had a problem to install the nightly rust with rustup. Because it does not seem compatible with rust installed with apt and I’d like to keep this rust for other stuff. And then I was too lazy to do it in a docker 🙂

Is there a reason to install with `apt` and not `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`?

**Comment by ashwinvis** [comment]
I have been using rustup in my elementaryOS laptop. It is a bit like pyenv for Python. You could switch the default toolchain easily. You can choose to use the rustup installation in the home directory (stable / nightly) or choose the system rust: https://rust-lang.github.io/rustup/installation/package-managers.html.

TLDR: some useful commands
```bash
$ rustup show
Default host: x86_64-unknown-linux-gnu
rustup home:  /home/avmo/.rustup

stable-x86_64-unknown-linux-gnu (default)
rustc 1.48.0 (7eac88abb 2020-11-16)
$ rustup default
stable-x86_64-unknown-linux-gnu (default)
$ rustup install nightly
....
$ rustup toolchain link system /usr
$ rustup toolchain list
stable-x86_64-unknown-linux-gnu (default)
nightly-x86_64-unknown-linux-gnu
system
$ rustup toolchain list
stable-x86_64-unknown-linux-gnu
nightly-x86_64-unknown-linux-gnu
system (default)
$ rustup show
Default host: x86_64-unknown-linux-gnu
rustup home:  /home/avmo/.rustup

installed toolchains
--------------------

stable-x86_64-unknown-linux-gnu
nightly-x86_64-unknown-linux-gnu
system (default)

active toolchain
----------------

system (default)
rustc 1.43.0
```
